tools/bazel-stamp-vars.js

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
```

Comment:
Why change this from a bash script?

Reply:
The bash script didn't seem to work on Windows. Filipe investigated this in https://github.com/bazelbuild/bazel/issues/4802, and I've just switched to a NodeJS script because it's easier to maintain, works on Windows and also is consistent to what `angular/angular` does.